radixdlt/src/main/java/com/radixdlt/consensus/SyncedStateComputer.java

```diff
@@ -38,6 +38,12 @@
 		 * @return the BFT data for the instruction
 		 */
 		VertexMetadata getVertexMetadata();
+
+		/**
+		 * Retrieve the BFT timestamp for the instruction
+		 * @return the BFT timestamp
+		 */
+		long getTimestamp();
```

Comment:
Don't think this is necessary for the syncedStateComputer to know as it is not related to synchronization

Reply:
OK, will remove.  Figured it would be required once we have time-based epochs, but it can go back in then, if required.